### ISSUE TITLE
WIN11 23H2 ISO from MSVLC April update yields Unknown Version

### ISSUE THREAD

**Issue by mattsmacblog** [alaurie/WimWitchFK#34]
## Expected Behavior

Version reports as 23H2 and not Unknown Version

## Current Behavior

![image](https://github.com/alaurie/WimWitchFK/assets/958733/224a68b9-9526-4d19-b2ab-5c38582119fb)

## Steps to Reproduce (for bugs)

1. Import the SW_DVD9_Win_Pro_11_23H2.5_64BIT_English_Pro_Ent_EDU_N_MLF_X23-73316.ISO, t
2. Tick .Net 3.5 and install.wim boxes. Hit Import
3. On the Source WIM tab, choose your newly imported WIM
4. Hit select, choose Enterprise 11 and it reports as Unknown Version (Build is 10.0.22621.3447)

## Context

I can't Make it So as I get a .Net 3.5 error about no matching .Net binaries for "Unknown Version".  If I make a folder of that name put the .Net binaries file in it, it works / continues on with no error.

## Your Environment

* Module version used: 4.0.1
* Operating System and PowerShell version: WIN 10 
PS VER Major  Minor  Build  Revision
-----  -----  -----  --------
5      1      19041  4291

**Comment by mattsmacblog** [comment]
TL;DR - Guess it needs updating to recognise this newer W11 23H2 build number...

**Comment by rootabagga** [comment]
I was hoping someone could help a noob like me.  I used to successfully use the old WimWitch, and I built a Wim file for Windows 11 22H2.  I would like to be able to create a wim file for Windows 11 23H2 using WimWitchFK.  I downloaded the zip file, and I extracted the files to C:\WimWitchFK-main.  I found a  ps1 file in the Public folder.  However, I wasn't able to run it using PowerShell, like I used to do in WimWitch.  It appeared to be the only PS1 file available.  Could someone let me know how I should best load WimWitchFK at this point?  Thank you so much!  Sincerely, rooty

**Comment by andrewljoy** [comment]
@rootabagga  You need to install from the powershell gallery then you can simply run winwitchfk from powersehll.

Yes this is an issue with the version number. needs to be updated

**Comment by act-GH** [comment]
Confirming behavior in 4.0.1

**Comment by jasasa** [comment]
You can add the new version by editing WWFunctions.ps1
Then find Function Get-WinVersionNumber, and add 
`'10\.0\.22621\.3737' { $buildnum = '23H2' }` 
under the Windows 11 version checks.
